lib/utils/parseCategoryApps.js

```diff
@@ -18,8 +18,23 @@ function checkIfCategoryIsReallyPaid (selectedCollection, allCategories) {
   return !firstApps[0].free;
 }
 
-function getParsedCluster (categoryObject, collection) {
-  const collections = {
+function getClusterMappings (collection) {
+  if (c.paths[collection].includes(c.clusters.new)) {
+    return {
+      2: {
```

Comment:
these objects look like they could be constants in the constants module or in a mappings module, rather than being generated dynamically inside this function.

Reply:
I created a new mapping modules for this.